content/working-groups/learning/minutes/2019.07.23-meeting.md

```diff
@@ -4,7 +4,7 @@ type: docs
 ---
 # Learning WG 2019-07-23 sync meeting
 
-## Rustc-guide strike force
+## rustc-dev-guide strike force
```

Comment:
Capital letter went off here

Reply:
This is consistent with how we are referring to it elsewhere when hyphens are used in the name.  Do you think that we need title casing?

Reply:
I actually prefer `rustc-dev-guide` too.